<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoleAuth</class>
 <widget class="QWidget" name="RoleAuth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QPushButton" name="doAuthBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
background-color: rgb(78, 160, 201);
border-radius: 5px; 
</string>
   </property>
   <property name="text">
    <string>角色授权</string>
   </property>
  </widget>
  <widget class="QTableView" name="role_view">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>521</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-width:1px;
border-color:rgb(190,190,190);
background-color: rgb(255, 255, 255);
border-radius: 6px; </string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="authCheckBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
background-color: rgb(78, 160, 201);
border-radius: 5px; 
</string>
   </property>
   <property name="text">
    <string>授权记录查看</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
